css/types/color.json

```diff
@@ -82,6 +82,39 @@
               "standard_track": true,
               "deprecated": false
             }
+          },
+          "mixed_type_parameters": {
+            "__compat": {
+              "description": "Mix <code>&lt;percentage&gt;</code> and <code>&lt;number&gt;</code> in parameters",
+              "support": {
+                "chrome": {
+                  "version_added": "111"
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "113"
+                },
+                "firefox_android": "mirror",
+                "ie": {
+                  "version_added": false
+                },
+                "oculus": "mirror",
+                "opera": "mirror",
+                "opera_android": "mirror",
+                "safari": {
+                  "version_added": "15"
+                },
```

Comment:
`color()` is the only color function that had support for the relaxed syntax from the start.

I am unsure if it's better to have a parallel between all color functions and explicitly list this for `color()` or to favor brevity and to omit it for `color()`.